### ISSUE TITLE
Solver.prototxt and train.prototxt are missing

### ISSUE THREAD

**Issue by smajida** [KaimingHe/deep-residual-networks#1]
Hi,

I think to finetune Residual networks on a different dataset using caffe, solver.prototxt and train.prototxt are needed, but they are missing and only deploy.prototxt has been provided.

-It would be good if you could put pretrained models for Pascal VOC benchmark plus prototxt files.
Best,
Seyedmajid

**Comment by KaimingHe** [comment]
The solver/train.prototxt files depend on your target dataset. For the VOC/COCO fine-tuning, the hyper-parameters can be found in our paper to set up these prototxt files.

**Comment by smajida** [comment]
for finetuning on a target dataset. I want to keep the settings you used in
solver.prototxt and train.prototxt to train on ImageNet dataset for object
recognition, unchanged except in training I will change the number of
outputs in the last layer and decrease the learning rate.

2016-02-04 17:13 GMT+01:00 Kaiming He <notifications@github.com>:
